packages/server-wallet/src/wallet/store.ts

```diff
@@ -94,6 +94,10 @@ export const Store = {
     return (await Channel.forId(channelId, tx))?.protocolState;
   },
 
+  getChannels: async function(tx: Transaction | undefined): Promise<Channel[]> {
```

Comment:
Maybe this should return a `Promise<ChannelResult[]>`, if the sole intention is to get a list of channels for the app.

A better signature would be `Promise<ChannelState[]>`, since there should be a function that converts a `ChannelState` to a `ChannelResult`.

Reply:
Yes you're right

Reply:
2729767ed